### PR TITLE
Support analytics (metadata and events) in the Tradeoff Analytics proxy

### PR DESCRIPTION
**Pull request by dolevdotan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "watson-developer-cloud",
-  "version": "0.10.3",
+  "version": "0.10.4",
```
Comment (germanattanasio) [inline package.json]: Versions are handle by a new release process. Please **don't modify it.**.
See [RELEASE.md](https://github.com/watson-developer-cloud/nodejs-wrapper/blob/master/RELEASE.md)

Comment (dolevdotan) [inline package.json]: Should I follow these steps? or are you going to do it?

Comment (germanattanasio) [inline package.json]: you should follow those steps.

Comment (nfriedly) [inline package.json]: @germanattanasio I would say that it should be your or me faccepting a PR and then ollowing the RELEASE.md steps. I don't think anyone else has permission to do the `npm publish` step.

**Comment by germanattanasio** [comment]
ups sorry I was trying to say that @dolevdotan should follow: https://github.com/watson-developer-cloud/nodejs-wrapper/blob/master/CONTRIBUTING.md :( my bad.

**Pull request closed without merging**